soh/src/code/z_player_lib.c

```diff
@@ -1851,6 +1862,24 @@ void Player_PostLimbDrawGameplay(PlayState* play, s32 limbIndex, Gfx** dList, Ve
                         Player_DrawHookshotReticle(play, this, RETICLE_MAX);
                     }
                 }
+            } else if (CVarGetInteger("gBoomerangReticle", 0) && (this->heldItemAction == PLAYER_IA_BOOMERANG)) {
+                if (Player_HoldsBoomerang(this) != 0) {
+                    MtxF sp44;
+                    s32 pad;
+
+                    if (LINK_AGE_IN_YEARS == YEARS_ADULT) {
+                        Matrix_RotateZYX(-31200, -9200, 17000, MTXMODE_APPLY);
+                    } else {
+                        Matrix_RotateZYX(-31200, -8700, 17000, MTXMODE_APPLY);
+                    }
+                    Matrix_Get(&sp44);
+
+                    if (Player_AimsBoomerang(this) != 0) {
+                        Matrix_Translate(500.0f, 300.0f, 0.0f, MTXMODE_APPLY);
+                        Player_DrawHookshotReticle(play, this, 38600.0f);
```

Comment:
It looks like the range for the reticle is a little bigger than the actual range of the boomerang, not that big of a deal but I think it'd be better if it's slightly smaller rather than slightly bigger to prevent annoying situations where you think the boomerang will hit but it doesn't.

Reply:
Corrected it! If I did the math right, 37000 is right for the aim spot, though technically it gets a little extra range on the return. I used 37000, since it makes the most sense with the actual reticle.